README.md

```diff
@@ -202,16 +202,18 @@ experiment Vue right away.
 #### Using Rails helpers in .vue files
 
 Rails helpers cannot be used in `.vue` files by default. To enable them, change
-the extension to `.vue.erb` and additionally amend the `test` in
-`config/webpack/loaders/vue.js` to also include `.vue.erb` files:
+the file extension to `.vue.erb` and additionally amend the `test` regex in
+`config/webpack/environment.js` to make `vue-loader` also include `.vue.erb` files:
 
 ```js
-# config/webpack/loaders/vue.js
+// config/webpack/environment.js
+const { environment } = require('webpacker')
 
-module.exports = {
-  test: /\.vue(\.erb)?$/,
-  ...
-}
+// Update an option directly
+const vueLoader = environment.loaders.get('vue')
+vueLoader.test = /\.vue(\.erb)?$/
```

Comment:
Should we make this the default `test` for the vue loader?
https://github.com/rails/webpacker/blob/3cd2e49b7943ddbea9da751a86b0ecdd80f2f61a/package/loaders/vue.js#L7

Reply:
Yeah we can do that but I guess then we would need to update all loaders to support `.erb` (if we want that out of the box)?

Reply:
We already do for ~half of the loaders. Might as well for all?

Reply:
Sounds good 👍  We also received this through many PR's and issues so makes sense 💯

Reply:
Cool. Want to make that change (and also add `tsx` to the typescript loader)?

Reply:
on it 👍